templates/secrets.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: v1
+kind: Secret
+metadata:
```

Comment:
Don't use stringData for storing the secrets. Also rename the file to `torqbit-secrets.yaml`